File: 884.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        Map<String, Integer> map = new HashMap<>();
        List<String> ans = new ArrayList<>();
        for(String word : s1.split(" ")) map.put(word, map.getOrDefault(word, 0) + 1);
        for(String word : s2.split(" ")) map.put(word, map.getOrDefault(word, 0) + 1);

        for(String word : map.keySet()) {
            if(map.get(word) == 1) ans.add(word);
        }
        return ans.toArray(new String[ans.size()]);
    }
}

File: 884.go
func uncommonFromSentences(s1 string, s2 string) []string {
	ans := []string{}
	m := make(map[string]int)
	word1, word2 := strings.Split(s1, " "), strings.Split(s2, " ")
	for _, word := range word1 {
		m[word]++
	}
	for _, word := range word2 {
		m[word]++
	}
	for k, v := range m {
		if v == 1 {
			ans = append(ans, k)
		}
	}
	return ans
}